.github/workflows/appimage.yml

```diff
@@ -4,7 +4,7 @@ on: [workflow_dispatch]
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
Same comment as above.

Reply:
oh, I see, let me revert the change and add code comment there